local-webhook/README.md

```
@@ -440,6 +444,61 @@ Differences from session routing, all deliberate:
+matched it, so two watches on one repo — "new issues" and "failing CI on
+`main`" — reach the spawn command as the same seven strings. A consumer that
+wanted to start a different worker for each had nothing to key on, and had to
+fall back to one setting for the whole box. That is the gap
+[defangdevs/agent-box#321](https://github.com/defangdevs/agent-box/issues/321)
+sat on: an agent profile (which CLI, which model, which effort) it could set
+box-wide but not per watch.
+
+A dispatch entry may carry `spawnConfig`: a flat map of strings, JSON-encoded
+into `LOCAL_WEBHOOK_SPAWN_CONFIG`.
+
+    python3 webhook.py subscribe 'github:defangdevs/*' --deliver-to subagent \
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)